.circleci/config.yml

```diff
@@ -401,3 +429,54 @@ jobs:
           name: Deploy release
           command: |
             bash ./build/upload.sh
+
+  test-render-macos-chrome:
+    macos:
+      xcode: 14.0.1 # macOS version 12.5.1 (Monterey)
+    working_directory: ~/mapbox-gl-js
+    steps:
+      - checkout
+      # TODO: implement cache
+      - attach_workspace:
+          at: ~/
+      - run: xcodebuild -version
+      - browser-tools/install-chrome
+      - run: yarn
+      - run: yarn run test-render
+      - store_test_results:
+          path: test/integration/render-tests
+      - store_artifacts:
+          path: "test/integration/render-tests/index.html"
+
+
+  test-render-windows-chrome:
+    executor:
+      name: win/default
+    working_directory: ~/mapbox-gl-js
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v4-yarn-{{ checksum "yarn.lock" }}
+      - attach_workspace:
+          at: ~/
+      # The browser-tools orb doesn't work on Windows, so we install chrome manually.
+      - run:
+          name: Installing Chrome on Windows
+          command: |
+            $uri = "https://dl.google.com/chrome/install/latest/chrome_installer.exe";
```

Comment:
Question: should we pin-it and manually update it, or always use latest? What's the approach taken for other browsers?

Reply:
On Linux and MacOS, browser-tools installs the latest release of both chrome and Firefox. IMO this is the behavior we want, along with testing the latest Chrome on Windows. We could also test older browser versions, though I think beta browsers are more important and though this might nontrivial to set up with Chrome.

From https://circleci.com/developer/orbs/orb/circleci/browser-tools:
> Install Google Chrome? Note: only the latest stable release can be installed, as Google does not maintain a public archive of previous releases.

Reply:
Sounds good to me, I wanted to confirm that's the behavior we wanted.